src/hipscat_import/catalog/resume_plan.py

```diff
@@ -129,15 +170,20 @@ def read_histogram(self, healpix_order):
     @classmethod
     def write_partial_histogram(cls, tmp_path, mapping_key: str, histogram):
         """Write partial histogram to a special intermediate directory"""
-        file_io.make_directory(
```

Comment:
[`map_to_pixels`](https://github.com/astronomy-commons/hipscat-import/blob/5ddb60c598028279f490a66184becc111d3b234f/src/hipscat_import/catalog/map_reduce.py#L114C8-L114C8) calls this method but it does not verify that the histogram's directory exists. Should we keep this make_directory call to make sure no error is thrown?